src/style.css

```diff
@@ -1 +1,86 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14.6px;
+  text-transform: uppercase;
```

Comment:
`html` is not the best place for these styles. Add them to the elements on which they belong.